packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -591,6 +598,7 @@ class RefreshIndicatorState extends State<RefreshIndicator> with TickerProviderS
                       valueColor: _valueColor,
                       backgroundColor: widget.backgroundColor,
                       strokeWidth: widget.strokeWidth,
+                        elevation: widget.elevation,
```

Comment:
Formatting nit: some extra space here. `elevation` needs to be lined up with the other proerties.